docs/core/deploying/deploy-with-cli.md

```diff
@@ -129,6 +129,21 @@ If you use the [example app](#sample-app), run `dotnet publish -f net6.0 -r win-
 > [!NOTE]
 > You can reduce the total size of your deployment by enabling **globalization invariant mode**. This mode is useful for applications that are not globally aware and that can use the formatting conventions, casing conventions, and string comparison and sort order of the [invariant culture](xref:System.Globalization.CultureInfo.InvariantCulture). For more information about **globalization invariant mode** and how to enable it, see [.NET Globalization Invariant Mode](https://github.com/dotnet/runtime/blob/main/docs/design/features/globalization-invariant-mode.md).
 
+### Configuring .NET install search behavior
+
+In .NET 9 RC1 and later versions, how the published executable looks for a .NET install can be configured via the [`AppHostDotNetSearch`](../project-sdk//msbuild-props.md#apphostdotnetsearch) and [`AppHostRelativeDotNet`](../project-sdk//msbuild-props.md#apphostrelativedotnet) properties.
```

Comment:
```suggestion
In .NET 9 RC1 and later versions, you can configure how the published executable looks for a .NET installation via the [`AppHostDotNetSearch`](../project-sdk//msbuild-props.md#apphostdotnetsearch) and [`AppHostRelativeDotNet`](../project-sdk//msbuild-props.md#apphostrelativedotnet) properties.
```

Reply:
The mention of RC1 makes more sense in "what's new". It doesn't make sense in topic-specific docs. We should just say ".NET 9 and later versions".

Reply:
> how the published executable looks

I think the common language for this might be "search paths" or something similar. I also think that this phrasing is more likely to be picked up by search engines.